sdk/gossip/client/client_test.go

```diff
@@ -120,6 +125,9 @@ func newClient(ctx context.Context, group *types.NotaryGroup, bootAddrs []string
 	if err != nil {
 		return nil, err
 	}
+
+	group.DagGetter = cli.DagGetter()
```

Comment:
I actually like this on notary group - I was thinking that in the wasm SDK "Community" should become the NotaryGroup too

Reply:
OK, cool. It felt a little weird in the all-nodes-in-the-same-process case, but that's only used in tests and some local dev scenarios.